RadarSDK/RadarLocationManager.m

```diff
@@ -240,10 +240,19 @@ - (void)requestLocation {
 }
 
 - (void)updateTracking {
-    [self updateTracking:nil];
+    [self updateTracking:nil fromInitialize:NO];
+}
+
+- (void)updateTrackingFromInitialize:(BOOL)fromInitialize {
```

Comment:
Wondering if this is an unusual name, given you pass the value `fromInitialize`? Wouldn't it be more standard to use `with`? Or just not pass any parameter here (pass-through would always be `YES`)?

Reply:
I agree that it's unusual, but it's what was recommended in the Shortcut. I don't think that `with` would make more sense, because it's supposed to indicate whether this function is being called _from_ the initializer. Perhaps `fromInitializer` would make more sense?

Reply:
my apologies for not setting expectations here - the shortcut ticket isn't meant to be 100% prescriptive in terms of exact names, style, and specific code changes, this one was particularly long since it had been talked about at length, and I wanted to capture all the details

I was thinking we could keep the fn name as `updateTracking` but just have the internal param name be `fromInitializer`, sort of how the `updateTracking` fn that took the location param was before this change

```
- (void) updateTracking:(BOOL)fromInitializer
```

how does that sound? open to feedback

Reply:
That's not possible in Objective-C, because I already declare a function with one parameter (`- (void)updateTracking:(CLLocation *)location`). Objective-C supports function overloading _only_ if the functions have different numbers of arguments, because it ignores parameter names when checking for overloads. The workaround is to change the _name_ of the function (which means you're no longer overloading it), which leads to the redundancy that @phil-brown points out on line 245.

<img width="811" alt="Screen Shot 2022-01-18 at 8 48 58 AM" src="https://user-images.githubusercontent.com/708510/149950240-3c03bcf1-1cf7-4247-bb35-399c8b3f9166.png">

Both of these functions have a signature of just `updateTracking(:)`, so that's not allowed. (It *is* allowed in Swift, which takes parameter names into account.)

Reply:
hmm, that function with one parameter is now going to take 2 after this PR, correct? At least from what I'm seeing on L254-L255
```
- (void)updateTracking:(CLLocation *)location
                 fromInitialize:(BOOL)fromInitialize
```

so the function signature I wrote out in my previous comment would be the only one with 1 param now

Reply:
Not quite. I left the original `(void)updateTracking:(CLLocation *)location` as-is so that all callers _other_ than the initializer could keep calling it, instead of changing all of them.

Reply:
ah I see, that's probably a cleaner approach - I'll defer to you then on what the new method / param name should be, I'm fine with `fromInitialize` / `fromInitializer` / or even something better you propose